src/server/graphql/mutations/changeTaskTeam.js

```diff
@@ -0,0 +1,97 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql';
+import getRethink from 'server/database/rethinkDriver';
+import ChangeTaskTeamPayload from 'server/graphql/types/ChangeTaskTeamPayload';
+import {getUserId, requireTeamMember} from 'server/utils/authorization';
+import shortid from 'shortid';
+import removeEntityKeepText from 'universal/utils/draftjs/removeEntityKeepText';
+import {TASK, TASK_INVOLVES} from 'universal/utils/constants';
+import publish from 'server/utils/publish';
+
+export default {
+  type: ChangeTaskTeamPayload,
+  description: 'Update a task with a change in content, ownership, or status',
```

Comment:
This description doesn't seem accurate. Can it say something like "Change the team a task is associated with"?